tools/cldr/lib/README.txt

```diff
@@ -80,3 +82,20 @@ org/unicode/cldr/cldr-api/0.1-SNAPSHOT/cldr-api-0.1-SNAPSHOT.jar
 Finally, if you choose to update the version number of the snapshot, then also
 update all the the pom.xml files which reference it (but this is unlikely to be
 necessary).
+
+Troubleshooting
+---------------
+
+While the Maven system should keep the CLDR JAR up to date, there is a chance
+that you may have an out of date JAR installed elsewhere. If you have any
+issues with the JAR not being the expected version (e.g. after making changes)
+then run the pruge step (above) again from this directory.
+
+This should re-resolve the current JAR snapshot from the repository in this
+directory. Having purged the Maven cache, you should now see something like:
+
+[exec] Downloading from XXXX: <url>
+[exec] Downloading from YYYY: <url>
+[exec] [INFO] Building jar: <path-to-icu-root>/tools/cldr/cldr-to-icu/target/cldr-to-icu-1.0-SNAPSHOT-jar-with-dependencies.jar
+
+Next time you build a Maven project (showing that it has re-fetched the JAR).
```

Comment:
Looks like a new and incomplete sentence. One idea was:

"Having purged the Maven cache, next time you build a maven project, you should now see something like:"

Small "n" to show an ongoing sentence probably looks too weird. Dunno, this is just a nit, you can leave it as is too.